pkgs/development/interpreters/pure/default.nix

```diff
@@ -3,14 +3,12 @@
 
 stdenv.mkDerivation rec {
   baseName="pure";
-  project="pure-lang";
-  version="0.66";
+  version="0.68";
   name="${baseName}-${version}";
-  extension="tar.gz";
 
   src = fetchurl {
-    url="https://bitbucket.org/purelang/${project}/downloads/${name}.${extension}";
-    sha256="42df6832476e8bee3a7ca179671284c1edd7bc82b71062fa0de62fd2117ee676";
+    url="https://github.com/agraef/pure-lang/releases/download/${name}/${name}.tar.gz";
```

Comment:
Can you rewrite this to use "fetchFromGitHub"?

Reply:
Yes, I think it's possible, but I'm not sure it is desirable.

The agraef/pure-lang repo is a "mega repo" containing Pure plus a bunch of Pure modules.  The maintainer packages up parts of the repo and uploads them using github's releases functionality.   If we used `fetchFromGitHub`, we would end up downloading a lot more than necessary just to build Pure.

Also, the release tarballs have had some autoconf, bison, etc. stuff run on them, so they have `configure` scripts, etc. ready to go.  I would need to update the build script to run this stuff instead.

If you think it's still worthwhile, let me know and I'll make the changes.